k8s/namespaces/reform-scan/blob-router-service/aat.yaml

```diff
@@ -15,22 +13,9 @@ metadata:
     repository.fluxcd.io/functional: java.functionaltests.image
```

Comment:
@adusumillipraveen don't these need updating to have the java prefix?

```suggestion
    repository.fluxcd.io/java.functional: java.functionaltests.image
```

Reply:
yes, they need to.

Reply:
do these annotations need to be in the aat.yaml file as well?

Reply:
If they are not in all envrionments.  In our case(draft-store), we had smoke tests enabled only on aat.

Reply:
also isn't it `java.functionaltests`/`java.smoketests`

Reply:
yes